src/anndata/_settings.py

```diff
@@ -432,7 +432,7 @@ def validate_type(val: V) -> None:
 
 def validate_zarr_write_format(format: int):
     validate_int(format)
-    if format != 2:
+    if format not in [2, 3]:
```

Comment:
always use set literals when checking for membership in a set of literals (strings, ints, …)